Dockerfile.template

```diff
@@ -130,6 +142,18 @@ RUN set -eux; \
 {{ ) else "" end -}}
 	; \
 	rm -rf /var/lib/apt/lists/*; \
+{{ if should_yjit then ( -}}
+	\
+	case "$yjit" in \
+		1) wget https://static.rust-lang.org/rustup/archive/1.25.1/${rustArch}/rustup-init; \
+		   echo "${rustupSha256} *rustup-init" | sha256sum --check --strict; \
+		   chmod +x rustup-init; \
+		   ./rustup-init -y --no-modify-path --profile minimal --default-toolchain $RUST_VERSION --default-host ${rustArch}; \
+		   rm rustup-init; \
+		   cargo --version; \
+		   rustc --version ;; \
```

Comment:
Is it necessary to display the `cargo` and `rustc` versions here if you're just going to delete them later?

Reply:
That's not (necessarily) to display them, but rather to smoke test that the downloaded binaries actually run correctly (so it can error out sooner than it will otherwise, probably deep inside the Ruby build).